lib/swoosh/email/unify.ex

```diff
@@ -1,13 +1,13 @@
-defmodule Swoosh.EmailHelpers do
+defmodule Swoosh.Email.Unify do
```

Comment:
Maybe this one should be `Swoosh.Email.Format` and the current `Swoosh.Email.Format` would be renamed to `Swoosh.Email.Render`?

What do you think?